# This message describes car data obtained from the CAN bus.
float64 speed
float64 steer_angle
float64 steer_rate
